evalml/model_understanding/graphs.py

```diff
@@ -1427,22 +1427,23 @@ def visualize_decision_tree(
     return source_obj
 
 
-def get_prediction_vs_actual_over_time_data(pipeline, X, y, dates):
+def get_prediction_vs_actual_over_time_data(pipeline, X, y, X_train, y_train, dates):
```

Comment:
Was this a bug? Or simply doing something different? If the latter, is there any functionality that we want to keep around?

Reply:
Got it, cool stuff !